lib/octokit/client/tokens.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module Octokit
+  class Client
+    # Method to check scopes and create_authorization required to test it
+    #
+    # @see https://developer.github.com/v3/oauth_authorizations/#oauth-authorizations-api
+    module Tokens
+      # Create an authorization for the authenticated user.
+      #
+      # You can create your own tokens, and only through
+      # Basic Authentication.
+      #
+      # @param options [Hash] A customizable set of options.
+      # @option options [Array] :scopes A list of scopes that this authorization is in.
+      # @option options [String] :note A note to remind you what the OAuth token is for.
+      # @option options [String] :note_url A URL to remind you what app the OAuth token is for.
+      # @option options [Boolean] :idempotent If true, will return an existing authorization if one has already been created.
+      # @option options [String] :client_id  Client Id we received when our application was registered with GitHub.
+      # @option options [String] :client_secret  Client Secret we received when our application was registered with GitHub.
+      #
+      # @return [Sawyer::Resource] A single authorization for the authenticated user
+      # @see https://developer.github.com/v3/oauth/#scopes Available scopes
+      # @see https://developer.github.com/v3/oauth_authorizations/#create-a-new-authorization
+      # @see https://developer.github.com/v3/oauth_authorizations/#get-or-create-an-authorization-for-a-specific-app
+      # @example Create a new authorization for user ctshryock's project Zoidberg
+      #  client = Octokit::Client.new(:login => 'ctshryock', :password => 'secret')
+      #  client.create_authorization({:scopes => ["public_repo", "gist"], :note => "Why not Zoidberg?", :note_url=> "https://en.wikipedia.org/wiki/Zoidberg"})
+      # @example Create a new OR return an existing authorization to be used by a specific client for user ctshryock's project Zoidberg
+      #  client = Octokit::Client.new(:login => 'ctshryock', :password => 'secret')
+      #  client.create_authorization({:idempotent => true, :client_id => 'xxxx', :client_secret => 'yyyy', :scopes => ["user"]})
+      def create_authorization(options = {})
```

Comment:
Why is this required to test `#scopes`? That method looks like it should work with any token and get its permissions.

Reply:
Could we not just call `#scopes` directly with a token, rather than creating one with `#create_authorization`?

Reply:
Welllllll of course we could do that! Forgive my slowness, and please take another look!